packages/calypso-e2e/src/lib/flows/start-site-flow.ts

```diff
@@ -167,4 +169,16 @@ export class StartSiteFlow {
 	async selectTheme( themeName: string ): Promise< void > {
 		await this.page.getByRole( 'button', { name: themeName } ).click();
 	}
+
+	/**
+	 * Confirms Theme selection by clicking the correct Continue button depending on viewport size.
+	 * themeContinueButton will be visible at viewports above 1080px.
+	 * themeContinueButtonSmall will be visible at viewports below 1080px.
```

Comment:
**comment:blocking** - turns out, the assumption here is not entirely correct!

As you can see here as an example, the Winkel theme shows the same `Continue` button regardless of the horizontal viewport width.

Desktop
![image](https://user-images.githubusercontent.com/6549265/226475291-a88008f7-8c7a-4c5d-9e2d-6f7c33d44d38.png)

Mobile
![image](https://user-images.githubusercontent.com/6549265/226477546-9daa6633-4543-470e-b46d-4dc2a349e55e.png)
----

Instead, the actual difference is whether the theme is FSE-enabled or not. You can see the difference with Vivre below, which is **not** a FSE theme.
![image](https://user-images.githubusercontent.com/6549265/226475612-3f4f4625-8de7-403a-a2fa-258e3c11f37b.png)

So in summary, we have two situations:
- theme is FSE: `Continue` button is shown to confirm the selected theme.
- theme is not FSE: `Start with ${themeName}` is button is shown to confirm the selected theme.

This has implications for the added selectors in `StartSiteFlow`, because we currently don't need to worry about mobile viewports.